<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionDelay</class>
 <widget class="QWidget" name="ActionDelay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="type">
       <item>
        <property name="text">
         <string>Action Delay</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Character Cooldown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Account Cooldown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Timer Extend</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblDelayID">
       <property name="text">
        <string>Delay ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="delayID">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblDuration">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="duration">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblActions">
       <property name="text">
        <string>Actions:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ActionList" name="actions" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionList</class>
   <extends>QWidget</extends>
   <header>ActionList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
